foundation/foundation-configuration-store/src/test/java/com/boozallen/aissemble/util/ConfigStoreInitTestHelper.java

```diff
@@ -1,154 +0,0 @@
-package com.boozallen.aissemble.util;
-
-/*-
- * #%L
- * aiSSEMBLE::Foundation::Configuration::Store
- * %%
- * Copyright (C) 2021 Booz Allen
- * %%
- * This software package is licensed under the Booz Allen Public License. All Rights Reserved.
- * #L%
- */
-
-import com.boozallen.aissemble.configuration.ConfigStoreInit;
-import com.boozallen.aissemble.configuration.store.Property;
-import io.quarkus.runtime.Startup;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-import org.yaml.snakeyaml.DumperOptions;
-import org.yaml.snakeyaml.Yaml;
-
-import javax.annotation.PostConstruct;
-import javax.enterprise.context.ApplicationScoped;
-import java.io.File;
-import java.io.FileWriter;
-import java.io.IOException;
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.LinkedHashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-import com.boozallen.aissemble.configuration.store.ConfigLoader;
-import javax.enterprise.inject.spi.CDI;
-import javax.enterprise.inject.Any;
-
-
-/**
- * ConfigStoreInit loads configuration properties as application starts up
- */
-@Startup
-@ApplicationScoped
-public class ConfigStoreInitTestHelper extends ConfigStoreInit {
-    private static final Logger logger = LoggerFactory.getLogger(ConfigStoreInitTestHelper.class);
-    public static Map<String, String> verification = new HashMap<>();
-    public static boolean skiploadingNotificationSent;
-    private static ConfigLoader configLoader;
-
-    @PostConstruct
-    protected void init() {
-        prepareTestEnv();
```

Comment:
Q: do we still need this for the test?

Reply:
The tests now use our config files that live in `src/test/resources` so this function was no longer needed for setup